src/docfx/lib/JsonUtility.cs

```diff
@@ -201,11 +201,14 @@ public static (List<Error>, T) ToObject<T>(JToken token, string docsetPath = nul
 
             void HandleError(object sender, Newtonsoft.Json.Serialization.ErrorEventArgs args)
             {
-                if (args.CurrentObject == args.ErrorContext.OriginalObject
```

Comment:
>args.CurrentObject == args.ErrorContext.OriginalObject [](start = 20, length = 54)

This is needed to handle the error once
[https://www.newtonsoft.com/json/help/html/SerializationErrorHandling.htm](https://www.newtonsoft.com/json/help/html/SerializationErrorHandling.htm)